src/domains/functions/data.tf

```diff
@@ -3,6 +3,11 @@ data "azurerm_key_vault" "common" {
   resource_group_name = local.rg_common_name
 }
 
+data "azurerm_key_vault" "io_com" {
+  name                = "io-p-itn-com-kv-01"
+  resource_group_name = "io-p-itn-com-rg-01"
```

Comment:
Use local project instead hardcoding the whole value.
es: `name = format("%s-itn-com-kv-01", local.project)`